$NetBSD$

Use the Linux frontend on BSD since the BSD frontend is incomplete.

Don't build the OSS driver twice.

--- Makefile.common.orig	2017-05-31 23:25:41.000000000 +0000
+++ Makefile.common
@@ -105,7 +105,7 @@ endif
 
 ifneq ($(findstring BSD,$(OS)),)
    BSD_LOCAL_INC += -I/usr/local/include
-   OBJ += frontend/drivers/platform_bsd.o
+   OBJ += frontend/drivers/platform_linux.o
 endif
 
 ifneq ($(findstring Darwin,$(OS)),)
@@ -370,9 +370,7 @@ endif
 
 ifeq ($(HAVE_OSS), 1)
    OBJ += audio/drivers/oss.o
-endif
-
-ifeq ($(HAVE_OSS_BSD), 1)
+else ifeq ($(HAVE_OSS_BSD), 1)
    OBJ += audio/drivers/oss.o
 endif
 
